src/main/java/strategy/FruitOperation.java

```diff
@@ -0,0 +1,8 @@
+package strategy;
+
+import dao.Storage;
+import model.FruitTransaction;
+
+public interface FruitOperation {
+    void execute(Storage storage, FruitTransaction fruitTransaction);
```

Comment:
Each operation has `Storage` as a class field
```suggestion
    void execute(FruitTransaction fruitTransaction);
```